wormhole-connect/src/hooks/useTrackTransfer.ts

```diff
@@ -19,9 +19,10 @@ type Props = {
 type ReturnProps = {
   isCompleted: boolean;
   isReadyToClaim: boolean;
+  receipt: routes.Receipt<AttestationReceipt> | undefined;
```

Comment:
We need to provide the current receipt back to the callers to prevent out-of-date UI states.